src/System.Collections/src/System/Collections/Generic/SortedSet.cs

```diff
@@ -1070,6 +1070,9 @@ public virtual void IntersectWith(IEnumerable<T> other)
             if (Count == 0)
                 return;
 
+            if (other == this)
+                return;
```

Comment:
yes but that's what all other SortedSet methods do. Should I make this one "inconsistent" ?